Weather App/UI/HomeVC/HomeView.swift

```diff
@@ -62,6 +57,12 @@ class HomeView : UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public func adjustFont(){
```

Comment:
Why is this needed?

Reply:
Not needed at all, it was supposed to be deleted I forgot to delete it, I deleted it now.